readthedocs/projects/views/private.py

```diff
@@ -374,8 +375,11 @@ def process_step(self, form):
                         "readthedocs.yml",
                     ]:
                         try:
+                            querystrings = (
+                                f"?ref={default_branch}" if default_branch else ""
```

Comment:
Do we want to pass `ref=` when there isn't a default branch? I'd probably do `querysetrings = ""` and then only set it when there's a `default_branch`

Reply:
> Do we want to pass `ref=` when there isn't a default branch?

No. In that case we just let GH to use the default branch from the repository.



> I'd probably do `querysetrings = ""` and then only set it when there's a `default_branch`

That is what the code I wrote does, doesn't? If no `default_branch` the `querystrings = ""`

Reply:
Oh, yea. I just read the logic wrong. 🙃